### PR TITLE
fixes things not pulling/moving properly 

### LINKED ISSUE #17052: Large Crystal Artifacts Cannot be Dragged wi
(WEB REPORT BY: anglosphere REMOTE: 172.93.109.202:7777)
# Revision
4219ed0cec997006eb2b336227040bc179d186f0

# Description
If a large crystal artifact is ctrl+left clicked, it cannot be dragged. 

https://imgur.com/a/D8hoY

Applies to both Green and Red/pink ones.  Otheir large artifacts seem not to be affected. 
# Steps to Reproduce
Find large crystal artifact
Ctrl+click
Try dragging it

# What you Expected
Find large crystal artifact 
Ctrl+click to drag it back to the outpost for research

# What Actually Happened
The artifact could not be dragged.

### LINKED ISSUE #17056: Cyborgs cannot be pulled
Cyborgs cannot be pulled. You can try but the pull breaks as soon as you move.

Possibly related to https://github.com/vgstation-coders/vgstation13/pull/17044
Almost certainly related to https://github.com/vgstation-coders/vgstation13/issues/17052

### LINKED ISSUE #17057: Windows and unsecured pipes cannot be pulled/thrown
Related to #17056 and #17052 
possibly related to #17044 
#### Description of issue
One direction windows, which normally are able to be pulled when unsecured, can no longer be pulled, however can be pushed. this is the same for unsecured piping sections, like a volume pump. It cannot be pulled, however, these items also cannot be thrown out of your tile. When thrown, the item lands at your feet on the same tile, and it hits your sprite.

### PR DESCRIPTION
**Pull request by MadmanMartian**
by giving them the proper arguments that were expected

Seems that adding a new argument to a byond-standard proc means you have to add the argument to every instance of trying to override it. Odd as you'd assume they would inherit from atom/movable, but that didn't seem to be the case.

closes #17056
closes #17057
closes #17052

### CODE REVIEW

**Comment by ShiftyRail** [comment]
Based.

File: code/game/objects/structures/vehicles/carts/mining_cart.dm [hunk 1]
```diff
@@ -3,7 +3,7 @@
 	icon = 'icons/obj/vehicles.dmi'
 	icon_state = "mining_cart"
 
-/*/obj/cart/mining_cart/Move()
+/*/obj/cart/mining_cart/Move(NewLoc, Dir = 0, step_x = 0, step_y = 0, glide_size_override = 0)
```
Comment (MadmanMartian) [inline code/game/objects/structures/vehicles/carts/mining_cart.dm:6]: lol

File: code/modules/mob/mob.dm [hunk 2]
```diff
@@ -1067,6 +1067,7 @@ var/list/slot_equipment_priority = list( \
 	set category = "IC"
 
 	if(pulling)
+		to_chat(world, "stop pulling called. pulling was [pulling], puller was [pulling.pulledby]")
```
Comment (MadmanMartian) [inline code/modules/mob/mob.dm]: FOUND A HITLER, HANG ON

**Comment by EdXbtkOEwGw** [comment]
T-thanks BYOND

File: code/game/objects/structures/vehicles/carts/mining_cart.dm (hunk 1 above)
Comment (Probe1) [reply code/game/objects/structures/vehicles/carts/mining_cart.dm:6]: What even

Comment (MadmanMartian) [reply code/game/objects/structures/vehicles/carts/mining_cart.dm:6]: They must have used the CTRL H replace all function

**Pull request merged**